storage/storage_dest.go

```diff
@@ -289,10 +294,13 @@ func (s *storageImageDestination) PutBlobPartial(ctx context.Context, chunkAcces
 	blobDigest := srcInfo.Digest
 
 	s.lock.Lock()
-	s.uncompressedOrTocDigest[blobDigest] = blobDigest
+	s.blobDiffIDs[blobDigest] = blobDigest
```

Comment:
That’s an unverified value.

From a quick check it seems to me that we don’t read it if `tocDigests` is set, so… I think we shouldn’t set it here. And insist on `tocDigest` being known.

Reply:
yes sorry, this is wrong, I'll drop this line